tree/20_cfe_basics/ncf_lib.cf

```diff
@@ -879,14 +879,14 @@ body classes classes_generic_return_boolean_list_two(x, y, boolean_prefix, true_
 # Define x and y prefixed/suffixed with promise outcome
 body classes classes_generic_two(x,y)
 {
-  !has_promiser_stack::
+#  !has_promiser_stack::
     promise_repaired => { "promise_repaired_$(x)", "$(x)_repaired", "$(x)_ok", "$(x)_reached" };
```

Comment:
You should drop those lines too since they duplicate what is done with has_promiser_stack below

Reply:
you are totally right !